### PR TITLE
Brake mode

### PR DESCRIPTION
**Pull request by Chvoong24**
works, util motor

### CODE REVIEW

**Review by CatonQ — APPROVED** [review]

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -32,6 +33,8 @@ public class Robot extends TimedRobot {
   public static final String LED = "LED";
   public static final String REACTION = "REACTION";
 
+  public static ReactionSubsystem m_ReactionSubsystem;
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/Robot.java:36]: This should be removed from here

File: src/main/java/frc/robot/GameRobotContainer.java [hunk 2]
```diff
@@ -97,6 +99,9 @@ private void configureBindings() {
         driverController.a().onTrue(new ChangeLEDColor(led, 255, 0, 255));
         driverController.rightBumper().onTrue(new ChangeLEDColor(led, 0, 0, 0));
         */
+
+        Shuffleboard.getTab("Climb").add("motor down", new MotorDown(m_Climbsubsystem, m_drivebase));
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/GameRobotContainer.java:103]: The changes go beyond only Brake Mode stuff.  Did we include more by accident?

File: src/main/java/frc/robot/subsystems/Climbsubsystem.java [hunk 3]
```diff
@@ -188,11 +164,11 @@ else if (downCheck && getBotRightLimit()) {
     }
 
     if (upCheck) {
-      setSpeed(-leftPIDvalue, rightPIDvalue);
+      setSpeed(-leftPIDvalue / 4, rightPIDvalue / 4);
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/subsystems/Climbsubsystem.java:167]: do we keep the divide by 4?

File: vendordeps/PathplannerLib.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
     "fileName": "PathplannerLib.json",
     "name": "PathplannerLib",
-    "version": "2024.2.3",
```
Comment (RobertTheBlair) [inline vendordeps/PathplannerLib.json:4]: I'm assuming the library updates are needed too, and you are rolling up many changes into this branch

**Review by RobertTheBlair — CHANGES_REQUESTED** [review]
Stray item in Robot.java, and wondering if the PR was intentionally changing things beyond brake mode.

**Pull request merged**